schema.sql

```diff
@@ -12,3 +12,42 @@ CREATE TABLE animals (
 
 ALTER TABLE IF EXISTS animals ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 5 );
 ALTER TABLE IF EXISTS animals ADD COLUMN species character varying(256);
+
+/*Create owners table*/
+
+CREATE TABLE owners (
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY,
+    full_name character varying(256) NOT NULL,
+    age integer NOT NULL,
+    PRIMARY KEY (id)
+);
+
+/*Create species table*/
+
+CREATE TABLE species (
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY,
+    name character varying NOT NULL,
+    PRIMARY KEY (id)
+);
+
+/*DELETE species column from animals table*/
+ALTER TABLE animals DROP COLUMN species;
+
+/*Add species_id & owner_id columns*/
+ALTER TABLE animals ADD COLUMN owner_id INTEGER;
+ALTER TABLE animals ADD COLUMN species_id INTEGER;
```

Comment:
-  Great job for adding the species_id column and the owner_id column to the animals table. However since these are foreign keys, we are supposed to reference the original tables which are `species` and `owners `respectively. Kindly modify these lines. **Hints**

ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
ALTER TABLE animals ADD COLUMN species_id INTEGER REFERENCES species(id);

**Please kindly modify before merging your PR since this is important**